website/src/pages/docs/index.mdx

```diff
@@ -170,3 +170,12 @@ server.listen(4000, () => {
 ```
 
 Visit [`http://localhost:4000/graphql`](http://localhost:4000/graphql) to see Yoga in action.
+
+<Callout type="info">
+**Get the best developer experience**
+
+GraphQL Code Generator and its server preset can help you build a type-safe and scalable GraphQL server.
+
+Learn about [GraphQL Code Generator](https://the-guild.dev/graphql/codegen/docs/getting-started/), then read [how to set up GraphQL server with the server preset](https://the-guild.dev/graphql/codegen/docs/guides/graphql-server-apollo-yoga-with-server-preset)
```

Comment:
A new user coming to GraphQL Yoga may not know what GraphQL Codegen is. So, I put the link to the Introduction page of Codegen here as well.